tests/Support/SupportCollectionTest.php

```diff
@@ -3601,7 +3601,7 @@ public function testReduceWithKeys($collection)
             'foo' => 'bar',
             'baz' => 'qux',
         ]);
-        $this->assertEquals('foobarbazqux', $data->reduceWithKeys(function ($carry, $element, $key) {
+        $this->assertEquals('foobarbazqux', $data->reduce(function ($carry, $element, $key) {
             return $carry .= $key.$element;
         }));
```

Comment:
1. Since we're still keeping `reduceWithKeys` around till 9.x (because it was already tagged), we shouldn't remove its test.
2. You can just add the `$key` to the other test. No need for 2 separate tests.
3. As discussed on that other PR, you move the `reduce` method into the `EnumeratesValues` trait.

Reply:
Will do, thanks for the comment